app-shells/loksh/loksh-7.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Linux port of OpenBSD's ksh"
+HOMEPAGE="https://github.com/dimkr/loksh"
+SRC_URI="https://github.com/dimkr/loksh/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+DEPEND="sys-libs/ncurses:0="
```

Comment:
If we're linking against it, we need it in RDEPEND too: RDEPEND="${DEPEND}" will suffice.

I'd also drop the "0" (it's implied).